src/api/Alert/index.ts

```diff
@@ -321,4 +323,17 @@ export class AlertAPI extends Request {
       params: params
     });
   }
+
+  /**
+   * Get all cases from alerts that a user has access to.
+   */
+  public getFullCaseList(this: Request<IFullCaseListResponse>, params: IGetFullCaseListParams) {
+    const encodedSortByParam = encodeURIComponent(JSON.stringify(params.sortBy));
```

Comment:
you do not need to use encodeURIComponent, this part is handled by the HTTP Client itself
only JSON.stringify is needed